CHANGELOG.rst

```diff
@@ -13,6 +13,8 @@ Backward-incompatible changes:
 Deprecations:
 ^^^^^^^^^^^^^
 
+- Deprecated ``OpenSSL.crypto.CRL``, ``OpenSSL.crypto.load_certificate_request``, ``OpenSSL.crypto.dump_certificate_request``.
```

Comment:
Changelog should link to the cryptography docs for this perhaps? What do you think?

Reply:
Ugh, probably.